### PR TITLE
docs: add external builder docs links

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

Makes the docs links for builders in djs clickable.

**Status and versioning classification:**

- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: packages/discord.js/src/structures/Message.js [hunk 1]
```diff
@@ -644,7 +644,7 @@ class Message extends Base {
    * @property {MessageAttachment[]} [attachments] An array of attachments to keep,
    * all attachments will be kept if omitted
    * @property {FileOptions[]|BufferResolvable[]|MessageAttachment[]} [files] Files to add to the message
-   * @property {MessageActionRow[]|MessageActionRowOptions[]} [components]
+   * @property {ActionRow[]|MessageActionRowOptions[]} [components]
```
Comment (imranbarbhuiya) [inline packages/discord.js/src/structures/Message.js:647]:
```suggestion
   * @property {ActionRow[]|ActionRowOptions[]} [components]
```

ref: https://github.com/discordjs/discord.js/blob/34120bba971132077d9140b588eeea7a8b4c889d/packages/discord.js/typings/index.d.ts#L4529

File: packages/discord.js/src/structures/Webhook.js [hunk 2]
```diff
@@ -134,7 +134,7 @@ class Webhook {
    * @property {FileOptions[]|BufferResolvable[]|MessageAttachment[]} [files] See {@link BaseMessageOptions#files}
    * @property {MessageMentionOptions} [allowedMentions] See {@link BaseMessageOptions#allowedMentions}
    * @property {MessageAttachment[]} [attachments] Attachments to send with the message
-   * @property {MessageActionRow[]|MessageActionRowOptions[]} [components]
+   * @property {ActionRow[]|MessageActionRowOptions[]} [components]
```
Comment (imranbarbhuiya) [inline packages/discord.js/src/structures/Webhook.js:137]:
```suggestion
   * @property {ActionRow[]|ActionRowOptions[]} [components]
```

File: packages/discord.js/src/structures/interfaces/TextBasedChannel.js [hunk 3]
```diff
@@ -62,7 +62,7 @@ class TextBasedChannel {
    * @property {MessageMentionOptions} [allowedMentions] Which mentions should be parsed from the message content
    * (see [here](https://discord.com/developers/docs/resources/channel#allowed-mentions-object) for more details)
    * @property {FileOptions[]|BufferResolvable[]|MessageAttachment[]} [files] Files to send with the message
-   * @property {MessageActionRow[]|MessageActionRowOptions[]} [components]
+   * @property {ActionRow[]|MessageActionRowOptions[]} [components]
```
Comment (imranbarbhuiya) [inline packages/discord.js/src/structures/interfaces/TextBasedChannel.js:65]:
```suggestion
   * @property {ActionRow[]|ActionRowOptions[]} [components]
```

**Review by imranbarbhuiya — CHANGES_REQUESTED** [review]

**Review by kyranet — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]
🎉

**Pull request merged**